src/plugin.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 let ReplaceSource = require('webpack-sources').ReplaceSource;
 let RawSource = require('webpack-sources').RawSource;
+let uniq = require('ramda').uniq;
```

Comment:
I've tried [ES6 Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set), [uniq](https://www.npmjs.com/package/uniq), [array-uniq](https://www.npmjs.com/package/array-uniq) and [lodash.uniq](https://www.npmjs.com/package/lodash.uniq), none work for array of objects, ramda is the one true way.